code/modules/client/preference_setup/loadout/loadout_suit.dm

```diff
@@ -64,19 +64,10 @@
 	gear_tweaks += new/datum/gear_tweak/path(hazard)
 
 /datum/gear/suit/hoodie
-	display_name = "hoodie selection"
+	display_name = "hoodie"
 	path = /obj/item/clothing/suit/storage/hooded/wintercoat/hoodie
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
 
-/datum/gear/suit/hoodie/New()
```

Comment:
These were added less than a month ago (the standard period where you can revert stuff), I do not see the reasoning for taking them out.

Reply:
This is still not past the revert period, and still lacks a reasoning.